bgpd/bgp_attr.c

```diff
@@ -2789,6 +2780,17 @@ bgp_attr_parse_ret_t bgp_attr_parse(struct peer *peer, struct attr *attr,
 		}
 	}
 
+	/*
```

Comment:
As this code is removed from bgp_attr_psid_sub() function, Similar code like below needs to be added in parse_test() function in test_mp_attr.c file. If anyone is using parse_test() function for their testing, it would be useful.

Reply:
Thank you for deep review

In current implementation, `parse_test()` function on `test_mp_attr.c` is for just attribute parsing, isn't it?
My modification code is fire when multiple attribute are presented, but this test can't verify that situation. In my understand. so I can't apply any change.?

If we want to check that "if BGP-update's safi isn't Labeled-unicast, label-indexl must be rewritten as INVALI_VAL", we need to add another unit-test or topotest. In current implementation, this unit-test doesn't guarantee that behavior. 

If my understand is wrong, please re-ask that.

BTW: prefix-sid binary definition in code isn't looks fine. I'll send another PR. to refactor that like follow
```
                "PREFIX-SID",
                "PREFIX-SID Test 1",
                {
-                       0x01, 0x00, 0x07,
-                       0x00, 0x00, 0x00, 0x00,
-                       0x00, 0x00, 0x02,
-                       0x03, 0x00, 0x08, 0x00,
-                       0x00, 0x0a, 0x1b, 0xfe,
-                       0x00, 0x00, 0x0a
+                       /* TLV[0] Latel-Index TLV */
+                       0x01,                    /* Type 0x01:Label-Index */
+                       0x00, 0x07,              /* Length */
+                       0x00,                    /* RESERVED */
+                       0x00, 0x00,              /* Flags */
+                       0x00, 0x00, 0x00, 0x02,  /* Label Index */
+
+                       /* TLV[1] SRGB TLV */
+                       0x03,                    /* Type 0x03:SRGB */
+                       0x00, 0x08,              /* Length */
+                       0x00, 0x00,              /* Flags */
+                       0x0a, 0x1b, 0xfe,        /* SRGB[0] first label */
+                       0x00, 0x00, 0x0a         /* SRBG[0] nb-labels in range */
                },
```

Reply:
Sure. Please go ahead with code modifications.

Please try to add a new topotest case to cover this.